### DISCUSSION TITLE
Sharing data between a Jukebox instance and a backend server instance

### DISCUSSION THREAD

**Discussion by nodje** [navidrome/navidrome#3544] [Q&A]
I want to be able to stream Navidrome music to my living room stereo.

I have a Navidrome instance on a backend server (online, no display, no sound), and use it with web and android client.
Works fine, I've got my playlists, favourite songs and all.

AFAIK I need a jukebox instance sitting next to my stereo, and this needs to be a dedicated Navidrome instance.
But of course, I want to keep using my playlists, favourite songs.
This would work if somehow I could mount the /data of the backend server instance to the Jukebox instance.
Not sure how to 2 live instance sharing the same data would work though.

That's the only option I see to be able to share my online server database.

Please let me know if I'm mistaken or if there's another way to do this

**Comment by metalheim** [comment]
I dont't think you can share the database between two navidrome instances that run at the same time.

I wouldn't advise running a full 2nd instance, just for Jukebox therefore. 
What you could do is consolidate to 1 server and configure the Jukebox output of this to pipe the audio to a network device that is nearby your speakers.
[That could be done via Snapcast f.e.](https://www.navidrome.org/docs/usage/jukebox/#the-mpvcmdtemplate--snapcast-integration)

Not sure how viable that is when your navidrome runs somewhere offsite. Might want to consider having 1 server that is actually in your home

**Reply by nodje** [reply]
thanks for the Snapcast idea, that'd probably work. Let's see how reliably.
My Navidrome server runs at home, just in the basement, not in my living room! ... so no worries on this side

**Reply by metalheim** [reply]
Then the snapcast idea should work (note: i dont use this myself, so i havent tested it), you just need some client device in your living room that acts as the snapcast receiver.
This could be a raspberry pi running snapcast or even an old android phone running https://github.com/badaix/snapdroid
